app/router.js

```diff
@@ -61,9 +61,9 @@ router.map(function() {
     this.route('applications');
   });
   this.route('explore');
-  this.route('my-tickets', function() {
```

Comment:
I am extremely sorry! I did it by mistake! I have updated the file. This will not happen again.